### DISCUSSION TITLE
contact for ordering spare parts

### DISCUSSION THREAD

**Discussion by frank-foerster** [robotology/community#585] [Q&A] [answered]
Hi all,

I wanted to order a replacement set of tendons for the iCub and a set of spare fingertips (the soft material), if possible.
Hence my question(s):
- does the "standard set" of replacement tendons still exist?
- Whom do I need to contact in order to order it?

Many thanks in advance

Frank

**Comment by pattacini** [comment, marked as answer]
Hi @frank-foerster 

To answer your questions:
>- does the "standard set" of replacement tendons still exist?
- Yes
>- Whom do I need to contact in order to order it?
- Andrea Derito @andreaderito, andrea.derito@iit.it

**Reply by frank-foerster** [reply]
HI @pattacini,
thank you very much! I'll drop Andrea an email.
